setup.py

```diff
@@ -55,19 +55,18 @@ def read(*names, **kwargs):
         # eg: 'keyword1', 'keyword2', 'keyword3',
     ],
     install_requires=[
-        'pynwb==1.1.2',
+        'pynwb==1.2.1',
         'appdirs',
         'h5py>=2.7.1',
         'nptdms',
         'numpy',
         'pandas>=0.20',
         'pyyaml',
-        'six',
         'tifffile',
     ],
     extras_require={
         ':python_version=="2"': ['enum34'],
```

Comment:
do we still need this line if we've stopped supporting python 2

Reply:
No, we shouldn't. We need to clear up some things now that we're no longer support Python 2, I'll add them to a relevant issue #29.